Resources/Prototypes/_LostParadise/Loadouts/Jobs/security.yml

```
@@ -1,37 +1,767 @@
+        - Security
+  items:
+    - WeaponSniperMosinRubber
+
+- type: loadout
+  id: LoadoutSecurityDB
+  category: JobsSecurity
+  cost: 2
+  requirements:
+    - !type:CharacterDepartmentTimeRequirement
+      department: Security
+      min: 36000 # 1 hours
```

yamllint

[warning] too few spaces before comment

(comments)